ngraph/core/src/op/roi_pooling.cpp

```diff
@@ -46,14 +46,6 @@ void op::ROIPooling::validate_and_infer_types()
         " and: ",
         coords_et);
 
-    NODE_VALIDATION_CHECK(
-        this,
-        feat_maps_et == coords_et,
```

Comment:
Hmm, should we change the reference implementation in this case? Because if I remember right it works with situations where `feat_maps_et == coords_et`